modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1460,6 +1460,113 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11553. If a product with variants follows a product without stock, only the product without stock shows the unavailable label"
+	@priority = 5
+	test CanViewProductWithStockAvailableWhenFollowingAZeroStockProduct {
```

Comment:
I think this name may be better 

test CanViewProductWithStockAvailableWhenItFollowsAProductWithoutStock